sources/packages/backend/libs/services/src/disbursement-schedule/disbursement-schedule-service.ts

```diff
@@ -365,18 +353,15 @@ export class DisbursementScheduleService extends RecordDataModelService<Disburse
   }
 
   /**
-   * Soft delete overawards records generated from reassessments and
-   * not related directly to any award. Overawards associated with some
-   * award must be rolled back only if the award is still on pending
-   * and this is taken care in a separated process.
+   * Soft delete overawards records generated from reassessments
+   * of this application.
```

Comment:
The comment could just add to mention that overawards are soft deleted if they exist for the given application.

Reply:
Changed.